### PR TITLE
Proposing Dapr to CNCF as an Incubation project

### PR DESCRIPTION
**Pull request by msfussell**
This PR contains a proposal to donate Dapr to the CNCF as an Incubation project.

Project Description: Dapr is a portable, event-driven runtime that makes it easy for developers to build resilient, stateless and stateful applications that run on the cloud and edge and embraces the diversity of languages and developer frameworks. 
Dapr codifies the best practices for building applications into open, independent, building blocks that enable you to build portable applications with any language and framework of your choice.

### CODE REVIEW

**Comment by iamalexmang** [comment]
🎉

**Comment by tomkerkhove** [comment]
Relates to https://github.com/dapr/dapr/issues/1849

Happy to see this, congrats @msfussell and team.

**Comment by AElharouny** [comment]
Congratulations 👏

**Comment by XavierGeerinck** [comment]
Amazing to see! 🤩

**Comment by zouyee** [comment]
congrats

**Comment by thombrn** [comment]
This is a great fit for both dapr and the CNCF

**Comment by nobodyiam** [comment]
Congratulations! 🎉
Dapr is a very promising project and we are happy to see this proposal!

**Comment by HowardvanRooijen** [comment]
This is excellent news. 🎉❤

**Comment by remiminnebo** [comment]
This is great! ❤

**Comment by skyao** [comment]
Congratulations! 🎉

This is a great news.

**Comment by khotyn** [comment]
Congrats!

**Comment by JervyShi** [comment]
Congratulations!

**Comment by rocklau** [comment]
We love it.

**Comment by ChenHaoHu** [comment]
great!

**Comment by yaoliu** [comment]
Congratulations!!

**Comment by ZhaoYis** [comment]
Congratulations

**Comment by WisWang** [comment]
Congratulations

**Comment by gauravhub** [comment]
Congrats!!

File: proposals/incubation/dapr.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+As part of the Dapr v1.0 release, Dapr focused on [production readiness](https://blog.dapr.io/posts/2021/02/17/announcing-dapr-v1.0/#release-highlights), in areas such as performance, security, High Availability (HA), conformance testing and end-to-end scenario testing.
+
+Dapr has had multiple, independent [security audits](https://docs.dapr.io/concepts/security-concept/#security-audit) from Cure53, CNCF approved security auditor.
+
+## Project logo in svg format
+
+All logos are available in https://github.com/dapr/docs/tree/v1.0/daprdocs/assets/icons
+
+The svg main logo is here: https://github.com/dapr/docs/blob/v1.0/daprdocs/assets/icons/logo.png
```
Comment (profporridge) [inline proposals/incubation/dapr.md:254]:
```suggestion
The svg main logo is here: https://github.com/dapr/docs/blob/v1.0/daprdocs/assets/icons/logo.svg
```

**Comment by thangchung** [comment]
Congrats! We're waiting for a long time

**Comment by tao12345666333** [comment]
Congratulations!

**Comment by wonderflow** [comment]
Congratulations!

**Comment by caniszczyk** [comment]
https://lists.cncf.io/g/cncf-toc/message/6293.

8/11
Lei Zhang: https://lists.cncf.io/g/cncf-toc/message/6303
Justin Cormack: https://lists.cncf.io/g/cncf-toc/message/6305
Erin Boyd: https://lists.cncf.io/g/cncf-toc/message/6321
Dave Zolotusky: https://lists.cncf.io/g/cncf-toc/message/6324
Ricardo Rocha: https://lists.cncf.io/g/cncf-toc/message/6325
Alena Prokharchyk: https://lists.cncf.io/g/cncf-toc/message/6329
Sheng Liang: https://lists.cncf.io/g/cncf-toc/message/6335
Davanum Srinivas: https://lists.cncf.io/g/cncf-toc/message/6336

+1 NB
Oleg Chornyi: https://lists.cncf.io/g/cncf-toc/message/6294
Matt Klein: https://lists.cncf.io/g/cncf-toc/message/6295
Tom Kerkhove: https://lists.cncf.io/g/cncf-toc/message/6296
Christopher Pisano: https://lists.cncf.io/g/cncf-toc/message/6302
Jim St. Leger: https://lists.cncf.io/g/cncf-toc/message/6304
Thomas Bern: https://lists.cncf.io/g/cncf-toc/message/6308
Lachlan Evenson: https://lists.cncf.io/g/cncf-toc/message/6309
Philippe Robin: https://lists.cncf.io/g/cncf-toc/message/6310
Will Tsai: https://lists.cncf.io/g/cncf-toc/message/6311
Hal Spang: https://lists.cncf.io/g/cncf-toc/message/6312
Bernd Verst: https://lists.cncf.io/g/cncf-toc/message/6313
Nick Greenfield: https://lists.cncf.io/g/cncf-toc/message/6314
Mark Fussell: https://lists.cncf.io/g/cncf-toc/message/6315
Sarah Novotny: https://lists.cncf.io/g/cncf-toc/message/6316
Lalit Fin: https://lists.cncf.io/g/cncf-toc/message/6317
Reyang: https://lists.cncf.io/g/cncf-toc/message/6318
Clemens Vasters: https://lists.cncf.io/g/cncf-toc/message/6320
Ricardo Aravena: https://lists.cncf.io/g/cncf-toc/message/6322
Darren Lambert: https://lists.cncf.io/g/cncf-toc/message/6323
Xavier Geerinck: https://lists.cncf.io/g/cncf-toc/message/6327
Paul Yuknewicz: https://lists.cncf.io/g/cncf-toc/message/6328
Ciprian Mocanu: https://lists.cncf.io/g/cncf-toc/message/6333
Robert Landers: https://lists.cncf.io/g/cncf-toc/message/6334

**Pull request merged**